types/jasmine/index.d.ts

```diff
@@ -290,9 +304,12 @@ declare namespace jasmine {
          * @since 3.5.0
          * @default undefined
          */
-        Promise?: Function | undefined;
+        Promise?: typeof Promise | undefined;
```

Comment:
Why not also add typeof PromiseLike?

Reply:
oh, I think it won't work. We need a function with constructor feature here, I was wondering of using old `Function` here to not create additional inline types to recreate `Promise` object.
This is how it was implemented, e.g. in `bluebird` (before they created `Promise`-based implementation by extending `PromiseLike` and providing constructor, etc.

Reply:
cool, I'll retract this part if required